apps/music/style/music.css

```diff
@@ -206,9 +211,16 @@ body.search-from-list-mode #views {
   transform: translateX(-100%) translateY(-5rem);
 }
 
+body.list-mode #views-list,
+body.picker-mode #views-list,
+body.search-from-list-mode #views-list {
```

Comment:
Same here, `#views-list` is also hidden when we entered the mode for searching from list.